cns/windows.Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /usr/local/src
 COPY . .
 RUN GOOS=windows CGO_ENABLED=0 go build -a -o /usr/local/bin/azure-cns.exe -ldflags "-X main.version="$VERSION" -X "$CNS_AI_PATH"="$CNS_AI_ID"" -gcflags="-dwarflocationlists=true" cns/service/*.go
 
-FROM mcr.microsoft.com/windows/servercore:${OS_VERSION}
```

Comment:
Are we going to lose any shell/util functionality by changing to nanoserver?

Reply:
we don't want shells or utils (the linux image is `FROM scratch`), so I don't care 😅

Reply:
Causing CNS failures `2023/12/21 01:42:24 [3600] Failed to check the existent of hns state path due to error Failed to find powershell executable`